Gemfile

```diff
@@ -8,6 +8,7 @@ gem 'bump'
 gem 'rack'
 gem 'rake'
 gem 'rspec', '~> 3.11'
+gem 'rubocop-capybara', github: 'rubocop-hq/rubocop-capybara'
```

Comment:
[note] This place should eventually need to refer to the following
https://github.com/rubocop/rubocop-capybara

Reply:
Ideally, this should not point to GitHub, but rather to a version published to RubyGems.